packages/docs/docs/350-examples.md

```diff
@@ -5,12 +5,12 @@
 Finds 0 or 1 record(s) in the `user` table where the `id` is in the list of ids.
 
 ```ts
-const q = from('users')
+const q = fromEcho('users')
```

Comment:
fromEcho shouldn't be part of teh examples as that's not an API in synthql it will just confuse people.